scripts/pylint_extensions_test.py

```diff
@@ -1569,3 +1569,192 @@ def test_divide_method_used(self):
 
         with self.checker_test_object.assertNoMessages():
             temp_file.close()
+
+
+class SingleLineCommentCheckerTests(unittest.TestCase):
+
+    def setUp(self):
+        super(SingleLineCommentCheckerTests, self).setUp()
+        self.checker_test_object = testutils.CheckerTestCase()
+        self.checker_test_object.CHECKER_CLASS = (
+            pylint_extensions.SingleLineCommentChecker)
+        self.checker_test_object.setup_method()
+
+    def test_invalid_punctuation(self):
+        node_invalid_punctuation = astroid.scoped_nodes.Module(
+            name='test',
+            doc='Custom test')
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""# Something/
+                """)
+        node_invalid_punctuation.file = filename
+        node_invalid_punctuation.path = filename
+
+        self.checker_test_object.checker.process_module(
+            node_invalid_punctuation)
+
+        with self.checker_test_object.assertAddsMessages(
+            testutils.Message(
+                msg_id='invalid-punctuation-used',
+                line=1
+            ),
```

Comment:
For readability, can this be defined before line 1600?

Reply:
@kevinlee12 did you mean something like this.
       `message = testutils.Message(
            msg_id='invalid-punctuation-used',
            line=1)

        with self.checker_test_object.assertAddsMessages(message):
            temp_file.close()`

Reply:
@kevinlee12 ptal!

Reply:
^ sorry, completely missed that comment.

Reply:
what you have is fine.